### ISSUE TITLE
Fix vue-template-compiler dependency problems

### ISSUE THREAD

**Issue by mikkonie** [bihealth/sodar-server#1971]
Another npm package bites the dust. `npm audit fix` offers a new version of `vue-template-compiler`. This breaks the Vue app, I'll post the exact crash below.

I'll try to see if there is a workaround to get all the dependencies to a point where they'll function, but I don't have my hopes up.

Using Vue2 and these old packages is like trying to put out forest fires with a squirt gun. We need to bite the bullet and do #994 soon, regardless of the effort required..

**Comment by mikkonie** [comment]
The app compiles after the test and seems to run file, at least in local dev mode. However, running tests results in the following:

```
Error: 

Vue packages version mismatch:

- vue@2.7.13 (/home/mikkopen/code/sodar/samplesheets/vueapp/node_modules/vue/dist/vue.runtime.common.js)
- vue-template-compiler@2.7.16 (/home/mikkopen/code/sodar/samplesheets/vueapp/node_modules/vue-template-compiler/package.json)

This may cause things to work incorrectly. Make sure to use the same version for both.
If you are using vue-loader@>=10.0, simply update vue-template-compiler.
If you are using vue-loader@<10.0 or vueify, re-installing vue-loader/vueify should bump vue-template-compiler to the latest.

    at Object.<anonymous> (/home/mikkopen/code/sodar/samplesheets/vueapp/node_modules/vue-template-compiler/index.js:10:9)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
    at Module.load (node:internal/modules/cjs/loader:1119:32)
    at Module._load (node:internal/modules/cjs/loader:960:12)
    at Module.require (node:internal/modules/cjs/loader:1143:19)
    at require (node:internal/modules/cjs/helpers:121:18)
    at Object.<anonymous> (/home/mikkopen/code/sodar/samplesheets/vueapp/node_modules/@vue/vue2-jest/lib/process.js:3:29)
    at Module._compile (node:internal/modules/cjs/loader:1256:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1310:10)
```

**Comment by mikkonie** [comment]
This will be fixed by #1972 once I can (hopefully) merge. Closing.
